sdk/python/resources/compute/attach_managed_spark_pools.py

```diff
@@ -0,0 +1,78 @@
+# import required libraries
+
+# Attach a Synapse Spark pool
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import SynapseSparkCompute
+from azure.identity import DefaultAzureCredential
+
+subscription_id = "<SUBSCRIPTION_ID>"
+resource_group = "<RESOURCE_GROUP>"
+workspace_name = "<AML_WORKSPACE_NAME>"
+
+ml_client = MLClient(
+    DefaultAzureCredential(), subscription_id, resource_group, workspace_name
+)
+
+synapse_name = "<ATTACHED_SPARK_POOL_NAME>"
+synapse_resource = "/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.Synapse/workspaces/<SYNAPSE_WORKSPACE_NAME>/bigDataPools/<SPARK_POOL_NAME>"
+
+synapse_comp = SynapseSparkCompute(name=synapse_name, resource_id=synapse_resource)
+ml_client.begin_create_or_update(synapse_comp).result()
+
+
+# Attach a Synapse Spark pool with system-assigned identity
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import SynapseSparkCompute, IdentityConfiguration
+from azure.identity import DefaultAzureCredential
+
+subscription_id = "<SUBSCRIPTION_ID>"
+resource_group = "<RESOURCE_GROUP>"
+workspace_name = "<AML_WORKSPACE_NAME>"
+
+ml_client = MLClient(
+    DefaultAzureCredential(), subscription_id, resource_group, workspace_name
+)
+
+synapse_name = "<ATTACHED_SPARK_POOL_NAME>"
+synapse_resource = "/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP>/providers/Microsoft.Synapse/workspaces/<SYNAPSE_WORKSPACE_NAME>/bigDataPools/<SPARK_POOL_NAME>"
```

Comment:
Can we please change this line to:
`synapse_resource = f"/subscriptions/{subscription_id}/resourceGroups/{resource_group}/providers/Microsoft.Synapse/workspaces/{workspace_name}/bigDataPools/{synapse_name}"`

Reply:
done